vm/devices/user_driver/src/vfio.rs

```diff
@@ -36,7 +36,7 @@ use zerocopy::AsBytes;
 use zerocopy::FromBytes;
 
 pub trait VfioDmaBuffer: 'static + Send + Sync {
-    /// Create a new dma buffer with the given `len` in bytes.
+    /// Create a new dma buffer with the given `len` in bytes. This buffer must be zeroed out.
```

Comment:
Why did this change?

Reply:
Oh, sorry, you're combining multiple changes into one PR.

In general, please don't do that.